nucypher/characters/lawful.py

```diff
@@ -684,8 +682,8 @@ def get_treasure_map_from_known_ursulas(self, network_middleware, map_identifier
                 raise _MapClass.NowhereToBeFound(f"Asked {len(self.known_nodes)} nodes, but none had map {map_identifier} ")
 
     def work_orders_for_capsules(self,
-                                 *capsules,
-                                 alice_verifying_key: UmbralPublicKey,
+                                 *message_kits,
+                                 alice_verifying_key: PublicKey,
```

Comment:
Why the API changes here from capsules to message_kits? I've read the description of the PR but I still don't get why the fact that a capsule is immutable should change anything at this level.

Reply:
Yes, you are right, this method only needs a capsule iterable.

Reply:
Fixed